spec/system/visit_index_page_spec.rb

```diff
@@ -13,4 +13,21 @@
     visit '/'
     expect(page).to have_content 'Manage certificates'
   end
+
+  it 'shows index page and successfully goes to next page' do
+    certificate = create(:msa_encryption_certificate)
+    msa_component = certificate.component
+    visit root_path
+    expect(page).to have_content 'Manage certificates'
+    click_link('Encryption certificate', match: :first)
+    expect(current_path).to eql view_certificate_path(msa_component.component_type, msa_component.id, msa_component.encryption_certificate_id)
+  end
+
+  it 'shows certificate expiring if under 30 days and IN USE if over 30 days' do
+    certificate = create(:msa_encryption_certificate)
```

Comment:
Non blocking, but could this and the one on `18` could be a `let` shared across the tests?